app/experimenter/nimbus-ui/src/components/PageEditOverview/index.tsx

```diff
@@ -65,14 +68,16 @@ const PageEditOverview: React.FunctionComponent<PageEditOverviewProps> = () => {
 
   return (
     <AppLayoutWithExperiment title="Overview" testId="PageEditOverview">
-      {({ experiment }) => {
+      {({ experiment, review }) => {
         currentExperiment.current = experiment;
+        refetchReview.current = review.refetch;
 
         return (
           <FormOverview
             {...{
               isLoading: loading,
               isServerValid,
+              isMissingField: review.isMissingField,
```

Comment:
This feels gross and I'm open to suggestions. I don't like that I'm passing all these review properties and functions around (including the ref above).

Reply:
🤷‍♂️  Maybe a `const { isMissingField } = review` before the return, in case we want to extract more review properties someday. But not a big deal